File: Python/Multiples of 3 and 5/multiples35.py
#  list all the numbers between 1 and 1000
# that are multiples of 3 or 5
result = 0
for i in range(1, 1000):
    if (i % 3 == 0 or i % 5 == 0):
#Find the sum of all the numbers
        result = result + i
print(result)

File: Javascript/Multiples of 3 and 5/multiples35.js
// list all the numbers between 1 and 1000
// that are multiples of 3 or 5
function multiples(number) {
  let sum = 0;
  for (let i = 0; i <= number; i++) {

    if (i % 3 === 0 || i % 5 === 0) {
      sum += i;
    }
  }
  return sum;
}

console.log( multiples(10) );
console.log( multiples(5) );
console.log( multiples(6) );

